packages/vouching/truffle-config.js

```diff
@@ -1,12 +1,12 @@
-require('babel-register')
-require('babel-polyfill')
-
 module.exports = {
   networks: {
-    local: {
+    local: { // Use `ganache-cli --deterministic`
```

Comment:
let's add a custom script in package.json instead and remove this comment
please also denote this in the readme